node/src/test/test-DataConsumer.ts

```diff
@@ -107,6 +107,20 @@ test('transport.consumeData() succeeds', async () => {
 	});
 }, 2000);
 
+test('transport.consumeData() with already used sctpStreamId rejects with Error', async () => {
+	await ctx.webRtcTransport2!.consumeData({
+		dataProducerId: ctx.dataProducer!.id,
+		sctpStreamId: 123,
+	});
```

Comment:
Please, create `const dataConsumer` here and assert that its `sctpStreamParameters.streamId` matches the given  `sctpStreamId`.